extensions/hp-extension/src/commandsModule.tsx

```diff
@@ -0,0 +1,99 @@
+const commandsModule = ({ servicesManager, commandsManager }) => {
+  const {
+    DisplaySetService,
+    ViewportGridService,
+    HangingProtocolService,
+  } = servicesManager.services;
```

Comment:
there are too many external services.

I would recommend to have some safe programming to prevent js exceptions, not only if there is the service but also the methods we are expecting to use